lib/tasks/publishing_api.rake

```diff
@@ -128,6 +128,29 @@ namespace :publishing_api do
     PublishingApiDocumentRepublishingWorker.new.perform(document.id)
   end
 
+  desc "Manually unpublish content with a redirect"
+  # This task is for unpublishing Whitehall managed content, where
+  # Whitehall has forgotten it is managing the content (as it often
+  # does).
+  task :unpublish_with_redirect, %i[content_id alternative_path locale] => :environment do |_, args|
+    args.with_defaults(locale: 'en')
+
+    document = Document.find_by(content_id: args[:content_id])
+    if document
+      puts "Document with this contnet ID exists: #{document}"
+      abort 'This Rake task is only for unknown content'
+    end
+
+    response = Services.publishing_api.unpublish(
+      args[:content_id],
+      type: 'redirect',
+      locale: args[:locale],
+      alternative_path: args[:alternative_path].strip
+    )
```

Comment:
Can we have a `:dry` version too?

```ruby
task :dry, %i[content_id alternative_path locale] => :environment do |_, args|
  # ...
  puts "This will unpublish #{args[:content_id]} and redirect it to #{args[:alternative_path]}" # or whatever
end
```

Reply:
Sure, I've added a `:dry_run` variant now, as well as the `:real` one.